packages/aragon-wrapper/src/identity/LocalIdentityProvider.test.js

```diff
@@ -0,0 +1,115 @@
+import test from 'ava'
+import sinon from 'sinon'
+import { EventEmitter } from 'events'
+
+import { LocalIdentityProvider } from './index'
+
+const ADDRESS_MIXED_CASE = '0x332462D19fC398189844E6F23685e19c21F5B265'
+const ADDRESS_LOWER_CASE = '0x332462d19fc398189844e6f23685e19c21f5b265'
+const SECOND_ADDRESS = '0x552462d19fc398189844e6f23685e19c21f5b200'
+const THIRD_ADDRESS = '0x7d77398078079B0D57ed872319F26D29B5405eb8'
+
+test.beforeEach(async t => {
+  t.context.localIdentityProvider = new LocalIdentityProvider()
+  await t.context.localIdentityProvider.init()
+  t.context.localIdentityProvider.clear() // clear because its storage is global
```

Comment:
> does the in-memory driver of localforage persist this global storage?

What do you mean by "persist" in this context. I think the in-memory driver relies on a "singleton" – https://github.com/localForage/localForage-memoryStorageDriver/blob/master/lib/localforage-memoryStorageDriver.js#L3-L16

Reply:
Yup, the singleton is what I meant by "persist". Makes sense, but if it didn't we could get away with not serializing these tests 😄

Reply:
I considered passing a `cacheKey` parameter to the `LocalIdentityProvider` constructor which would allow us to run tests in parallel but I couldn't think of a real use-case besides the tests. 🤔

Reply:
Totally agree, no reason to introduce this simply to make the tests slightly faster 👍